distribution/lib/Standard/Base/0.0.0-dev/src/Visualization/Table_Viz_Data.enso

```diff
@@ -1,17 +1,32 @@
 from project.Data.Json import JS_Object
 
 ## Type for generating the visulaisation data for the "Table" visualisation
+
+   IMPORTANT - The types here must be kept in sync with those in TableVisualisationTypes.ts
+
    if you want your type to support this visualisation type you should provide a conversion to this type
    e.g. ## PRIVATE
               Table_Viz_Data.from (that:MyType) =
 type Table_Viz_Data
-    ## For now provide the json directly. In the future this type will be responsible for the json creation
+    ## Temporary constructor for viz types that have not yet moved to this model
     Value json:JS_Object
 
-    ## Create a new Table_Viz_Data
-    new json:JS_Object -> Table_Viz_Data = Table_Viz_Data.Value json
+    ## Table Viz for errors
+    Error error_display_text
+
+    ## Table Viz for a single column of clickable items
+    SingleColumnOfActions js_value data title tooltip child_node_action
```

Comment:
Probably would be better to rename to `Single_Column_Of_Actions` to be consistent with naming scheme.